src/main_controller.ts

```diff
@@ -185,18 +184,20 @@ export class MainController implements vscode.Disposable {
             }
         }
 
-        let neovimPath = config.neovimPath;
-        // Only try to find nvim if the path is the default one
-        // And if we are not using WSL
-        if (neovimPath === "nvim" && !config.useWsl) {
```

Comment:
Why skip this for WSL?

Reply:
The purpose of our useWsl is to use nvim within WSL, which is different from remote development. findNvim is not functioning properly.

Reply:
Ok, I see that this is blocked by https://github.com/neovim/node-client/issues/430